src/main.js

```diff
@@ -53,40 +60,75 @@ function randomizePoster() {
   secondDescriptor.innerText = currentCover.tagline2;
 }
 
-function createCustomCover() {
-  currentCover = new Cover(
-    userCover.value,
-    userTitle.value,
-    userDescriptorOne.value,
-    userDescriptorTwo.value
-  );
-  // covers.unshift(userCover.value);
-  // titles.unshift(userTitle.value);
-  // descriptors.unshift(userDescriptorOne.value);
-  // descriptors.unshift(userDescriptorTwo.value);
-}
-
-function displayCustomCover() {
-  cover.src = currentCover.cover;
-  title.innerText = currentCover.title;
-  firstDescriptor.innerText = currentCover.tagline1;
-  secondDescriptor.innerText = currentCover.tagline2;
+function noDuplicates() {
```

Comment:
The function name here is a little misleading. We want there to be no duplicates but the function itself is returning a `boolean` value. It might be better to name the function something like `hasDuplicates` - which returns `true` or `false`

Reply:
Totally! Thank you for the feedback

Reply:
@djavanm That makes sense.  Thanks for the feedback!